src/test/java/org/benetech/mathshare/model/mother/ProblemSetRevisionUtils.java

```diff
@@ -5,10 +5,8 @@
 
 public abstract class ProblemSetRevisionUtils {
```

Comment:
change Utils to Mother or Builder. 'Utils' name suggest a different purpose of these classes

Reply:
According to https://github.com/SolDevelo/static-code-analysis-rules, all classes with only static methods should be named ...Utils or ...Helper

Reply:
I agree with @pawelcieszko, Mother seems to fit better here.